src/plugins/dashboard/public/plugin.tsx

```diff
@@ -86,26 +89,44 @@ export class DashboardEmbeddableContainerPublicPlugin
         }))
       );
     }
+
+    const getStartServices = async () => {
+      const [coreStart, deps] = await core.getStartServices();
+
+      const useHideChrome = () => {
```

Comment:
I suspect this is going to be re-used in visualize and discover. 
Maybe we want to put this in kibana_react?